testing/regress/ecl/genjoin.ecl

```diff
@@ -25,8 +25,10 @@ jrec := RECORD,MAXLENGTH(100)
     UNSIGNED1 i;
     STRING3 lstr;
     STRING3 rstr;
-    UNSIGNED1 c;
```

Comment:
I don't know whether it's a historical remnant or not... Try running genjointest.pl and see how different the generated tests are from what is checked in.